shell/app/node_main.cc

```diff
@@ -221,6 +171,29 @@ int NodeMain(int argc, char* argv[]) {
       return result->exit_code();
     }
 
+#if BUILDFLAG(IS_LINUX)
+    // On Posix, initialize crashpad after Nodejs init phase so that
```

Comment:
```suggestion
    // On Linux, initialize crashpad after Nodejs init phase so that
```

macOS is POSIX(-ish) :)

Reply:
Addressed in [ecd98e8](https://github.com/electron/electron/pull/36460/commits/ecd98e89805fb167b2074a91f3a49e0c4aa68c5a)

Reply:
Oh I did mean this comment for macOS as well, since the signal reset code is behind `__POSIX__` https://github.com/nodejs/node/blob/ed3604cd64765cb6dfb0b7160ce1b829666d3194/src/node.cc#L418-L434

Reply:
Oh nice! I'll re-add the original comment.

Reply:
but we do this on Windows too..?

Reply:
I'll leave the comment as is for now, since this is explicitly being used for Linux, and maybe we can re-evaluate whether or not to expand it.

Reply:
> but we do this on Windows too..?

The delay on Windows is not needed, I just tried to keep the `#if #else` together since there is no downside to delaying in windows. Also, for macOS crashpad uses mach ports to listen for exceptions in the process. So the signal handler reset in Nodejs will not affect macOS.

We can make the following change, move windows and macOS code to original position. Only delay on linux and change the comment to reflect it as well.

Reply:
I think the logic makes sense as it is, since there's no downside/significant changes to move the logic down for Mac and Windows as well 🙂